modules/kuberay/variables.tf

```diff
@@ -0,0 +1,67 @@
+variable "folder_id" {
+  description = "The ID of the folder that the Bastion should be deployed to."
+  type        = string
+  default     = null
```

Comment:
If `null` value for variable breaks module, better avoid `default` property for it.